go/vt/vtctl/grpcvtctldserver/server.go

```diff
@@ -2259,24 +2260,26 @@ func (s *VtctldServer) GetTopologyPath(ctx context.Context, req *vtctldatapb.Get
 	span, ctx := trace.NewSpan(ctx, "VtctldServer.GetTopology")
 	defer span.Finish()
 
-	// handle toplevel display: global, then one line per cell.
-	if req.Path == "/" {
```

Comment:
nit: proto fields are all public by design, and it is fine to access them directly. There is no need to change all accesses to use a getter instead.

Reply:
I've gotten into the habit of using the proto getters as they are nil safe. In this case, for example, if req is nil it will return "" rather than have a nil pointer error.